src/main/xar-resources/data/upgrading/upgrading.xml

```diff
@@ -1,209 +1,234 @@
+                    Emergency Backup Tool).</para>
+                <para>Normally this report just contains the list of collections which were backed up. If
+                    you see any warnings or errors, it is even more  important to perform a “full
+                    restore” of your backed-up data.</para>
+            </listitem>
+            <listitem>
+                <para>Stop the existing installation of eXist.</para>
+            </listitem>
+            <listitem>
+                <para>Install the new version of eXist. DO not install one eXist version over
+                    another one. Always install the new version into a fresh new directory.</para>
+            </listitem>
+            <listitem>
+                <para>Perform a full <link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="backup.xml#restore">restore</link> of the backup onto the new version of
+                    eXist.</para>
+            </listitem>
         </orderedlist>
-        <para>Once the backup is complete, you should check the backup report in the
-            <code>export</code> directory within either the <code>data</code> directory (for backups created via
-          the Dashboard or Xquery) or <code>$EXIST-HOME</code> (for backups created by the
-          Emergency Backup Tool).</para>
-        <para>Normally this report just contains the list of collections which were backed up. If
-          you see any warnings or errors, it is even more  important to perform a “full
-          restore” of your backed-up data.</para>
-      </listitem>
-      <listitem>
-        <para>Stop the existing installation of eXist.</para>
-      </listitem>
-      <listitem>
-        <para>Install the new version of eXist. DO not install one eXist version over
-          another one. Always install the new version into a fresh new directory.</para>
-      </listitem>
-      <listitem>
-        <para>Perform a full <link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="backup.xml#restore">restore</link> of the backup onto the new version of
-          eXist.</para>
-      </listitem>
-    </orderedlist>
-  </sect1>
-
-  <!-- ================================================================== -->
-
-  <sect1 xml:id="compatibility-history">
-    <title>History of Non-compatible Upgrades</title>
-
-    <para>This is a list of binary non-compatible releases for quick reference.</para>
-    <para>If a non-incremental upgrade across multiple non-compatible releases is necessary,
-      foir instance from 1.4 to 3.6, we strongly recommend to first check past release notes to understand
-      how breaking changes might affect you.</para>
+    </sect1>
+    
+    <!-- ================================================================== -->
+    
+    <sect1 xml:id="upgrading-from-source">
+        <title>Upgrading from Source</title>
+        <orderedlist>
+            
```

Comment:
needs an intro, to make clear that it only applies to folks building and running from source